app/views/hyrax/collections/show.html.erb

```diff
@@ -1,73 +1,89 @@
-                <% else %>
-                    <a href="<%= lr[:linkurl] %>">
-                      <img alt="<%= lr[:alttext] %>" src="<%= lr[:file_location] %>" />
-                    </a>
-                <% end %>
-
-            <% end %>
-          </div>
-      <% end %>
-
-      <% unless @presenter.total_viewable_items.blank? %>
-          <div class="hyc-bugs">
-            <div class="hyc-item-count">
-              <b><%= @presenter.total_viewable_items %></b>
-              <%= 'Item'.pluralize(@presenter.total_viewable_items) %></div>
 
-            <% unless @presenter.creator.blank? %>
-                <div class="hyc-created-by">Created by: <%= @presenter.creator.first %></div>
-            <% end %>
 
-            <% unless @presenter.modified_date.blank? %>
-                <div class="hyc-last-updated">Last Updated: <%= @presenter.modified_date %></div>
-            <% end %>
-          </div>
+<div class="hyc-container" itemscope itemtype="http://schema.org/CollectionPage">
+  <div class="row">
+    <div class="hyc-blacklight hyc-bl-results">
+      <div class="row">
+      <div class="col-md-2"></div>
+      <% @member_docs[0,4].each_with_index do |doc, i| %>
+        <div class="masonry document text-center col-xs-3 col-md-2">
+          <%= render 'showcase_tile',
+              title: doc.title_or_label,
+              index: i,
+              count: @member_docs.length,
+              link: [hyrax, doc],
+              img: render_thumbnail_tag(doc, {alt: ''}, { suppress_link: true }) %>
       <% end %>
-
       </div>
-
     </div>
   </div>
+  <div class='row'>
+    <div class="col-sm-1"></div>
+    <div class="col-sm-4"><hr class="apple-green-hr"></div>
+    <div class="col-sm-2" ><button class='btn btn-primary collection-landing-button'>View All</button></div>
+    <div class="col-sm-4"><hr class="apple-green-hr"></div>
+    <div class="col-sm-1"></div>
+  </div>
 
-  <div class="row hyc-body">
-    <div class="col-md-8 hyc-description">
-      <%= render 'collection_description', presenter: @presenter %>
-
-      <% if @presenter.collection_type_is_nestable? && @presenter.total_parent_collections > 0 %>
-          <div class="hyc-blacklight hyc-bl-title">
-            <h2>
-              <%= t('.parent_collection_header') %> (<%= @presenter.total_parent_collections %>)
-            </h2>
-          </div>
-          <div class="hyc-blacklight hyc-bl-results">
-            <%= render 'show_parent_collections', presenter: @presenter %>
-          </div>
-      <% end %>
+  <div class="row">
+    <div class="col-sm-5"></div>
+      OR, SEARCH THIS COLLECTION
+  </div>
 
+  <!-- Search bar -->
+  <div class="hyc-blacklight hyc-bl-search hyc-body row">
+    <div class="col-sm-4"></div>
+    <div class="col-sm-4">
+      <%= render 'search_form', presenter: @presenter, url: hyrax.collection_path(@presenter.id) %>
     </div>
+    <div class="col-sm-4"></div>
+  </div>
+
+  <div class='row'>
```

Comment:
We use these rows a few places in the wire frames. Consider them for a refactor of template files